app/views/MultipleQuestionView.scala.html

```diff
@@ -80,6 +81,27 @@ <h1 class="govuk-heading-l govuk-!-margin-bottom-3">@renderMessage(viewmodel.hea
     )
 }
 
+
+@numeric(index: Int, width: Option[InputWidth], field: QuestionField) = {
```

Comment:
Indentation looks weird, all spaces were removed.

Reply:
Apologies, My intelliJ is the community edition it does not like this file format haha